torch/nn/parallel/distributed.py

```diff
@@ -290,7 +305,23 @@ def _check_default_group(self):
                                "init_process_group and have not passed "
                                "process_group argument to DDP constructor")
 
+    def _check_previous_reduction(self):
+        if not self.training:
+            return
+        if self.check_previous_reduction is False:
+            self.check_previous_reduction = True
```

Comment:
I thought this var is unused but see it's used to toggle to True once and then keep it that way. Could you add a comment here along the lines of: check_previous_reduction will be False in the first iteration and is then toggled to True for all future iterations.